cars.html

```diff
@@ -64,6 +64,7 @@
 	        }
 	    };
 	}
+	var fluffy = new Dog("Bobik", "alabay", 45);
 	var fluffy = new Dog("Fluffy", "Poodle", 30);
 	var spot = new Dog("Spot", "Chihuahua", 10);
 	var dogs = [meow, whiskers, fido, fluffy, spot];
```

Comment:
```suggestion
	var fluffy = new Dog("Sharick", "alabay", 45);
	var fluffy = new Dog("Fluffy", "Poodle", 30);
	var spot = new Dog("Spot", "Chihuahua", 10);
	var dogs = [meow, whiskers, fido, fluffy, spot];
``` I offer you to change name of your dog